### DISCUSSION TITLE
Whole result repeated

### DISCUSSION THREAD

**Discussion by wbz5087** [openai/whisper#1280] [Q&A]
I use medium model to do transcribe task. Then I found some results were repeated. I tried to set a lower compression_ratio_threshold and set temperature to 0, but the problem still exists.
The I print the result details, I found that if it has two segments, the final text will combine the results from each segment and repeat twice.  As shown in pictures, the red one shows how it repeat, and the blue one shows it has two segments.
How can I slove this problem? 
![1](https://user-images.githubusercontent.com/12483092/234180114-2ebb0949-96ea-418f-8c10-93fadf3e8710.png)
![2](https://user-images.githubusercontent.com/12483092/234180120-cb16c375-eb24-44d4-8eb0-335c4e124f3d.png)

**Comment by jlmaoju** [comment]
Same issue
